% Chapter 02
\chapter{Alternative polyadenylation mediates genetic regulation of gene expression}\label{ch:tb}
\section[Abstract]{Abstract\footnotemark}


%\ref{fig:response-eqtl}. 

Little is known about co-transcriptional or post-transcriptional regulatory mechanisms linking noncoding variation to variation in organismal traits. To begin addressing this gap, we used 3' Seq to study the impact of genetic variation on alternative polyadenylation (APA) in the nuclear and total mRNA fractions of 52 HapMap Yoruba human lymphoblastoid cell lines. We mapped 602 APA quantitative trait loci (apaQTLs) at 10\% FDR, of which 152 were nuclear specific. Effect sizes at intronic apaQTLs are negatively correlated with eQTL effect sizes. These observations suggest genetic variants can decrease mRNA expression levels by increasing usage of intronic PAS. We also identified 24 apaQTLs associated with protein levels, but not mRNA expression. Finally, we found that 19\% of apaQTLs can be associated with disease. Thus, our work demonstrates that APA links genetic variation to variation in gene expression, protein expression, and disease risk, and reveals uncharted modes of genetic regulation.

\footnotetext{Citation for chapter: Mittleman BE, Pott S, Warland S, Zheng T, Mu Z, Kaur M, Gilad Y, and Li YI. Alternative polyadenylation mediates genetic regulation of gene expression. 2020 June 25; eLife 2020;9:e57492;  DOI: 10.7554/eLife.57492 }


\clearpage


\section{Introduction}\label{ch02-introduction}

Most genetic variants associated with complex traits are noncoding, suggesting that inter-individual variation in gene regulation plays a dominant role in determining phenotypic outcome. To investigate the function of trait-associated variants identified using genome-wide association studies (GWAS), studies have used regulatory quantitative trait loci (QTL) mapping to associate GWAS loci with variation in mRNA expression levels, DNA methylation levels, and other molecular phenotypes. Although many GWAS loci affect mRNA expression levels (i.e. are eQTLs), several recent discoveries highlight the pressing need for a better understanding of the genetic control of gene regulation, beyond that of just mRNA expression levels. For example, one recent study \citep{chun_limited_2017} found that the majority of autoimmune GWAS loci do not appear to affect mRNA expression levels. Two other studies observed that many genetic variants that affect protein expression levels (pQTLs) do not affect mRNA expression levels \citep{battle_genomic_2015, chick_defining_2016}. Specifically, Battle and colleagues found that about half of the cis-pQTLs they identified in human LCLs (146 out of 278, 52\%) did not appear to impact gene expression levels in the same lymphoblastoid cell lines (LCLs) \citep{battle_genomic_2015}. Altogether these findings indicate that there may be unknown or understudied regulatory mechanisms that link genetic variation to complex traits, and that these mechanisms are independent of changes in the amplitude of mRNA expression levels. Moreover, even when a disease-associated variant impacts mRNA expression levels, the mechanisms by which expression is affected is often unclear. Indeed, a third of all eQTLs identified in human LCLs are not associated with variation in chromatin as measured using assays for chromatin accessibility or for modification levels of several histone marks \citep{li_rna_2016}. These observations raise the possibility that understudied regulatory mechanisms mediate the effect of a substantial number of genetic variants on gene expression level.

One such understudied mechanism is alternative polyadenylation (APA). Well over half of all human protein coding genes encode multiple polyadenylation sites (PAS), resulting in the production of diverse mRNAs with alternative termination sites \citep{tian_alternative_2017, mayr_evolution_2016, shi_alternative_2012}. Unlike alternative mRNA splicing, which leads to changes in splice site selection, APA leads to changes in the transcript termination site, often resulting in 3' untranslated regions (UTRs) with different lengths. As 3' UTRs are densely packed with regulatory elements that impact mRNA stability, miRNA binding, and mRNA localization (reviewed in \citep{mayr_regulation_2017,tian_alternative_2017}), genetic control of APA may be a key mechanism by which genetic variants impact gene regulation, including mRNA expression levels, without affecting chromatin-level phenotypes such as promoter or enhancer activity. Moreover, proteins translated from different APA isoforms may differ in length and protein-protein interactions, and these differences can impact cellular phenotype. For example, globally increased usage of intronic PAS has been shown to increase risk for multiple myeloma and chronic lymphocytic leukemia \citep{lee_widespread_2018, singh_widespread_2018} through the translation of truncated mRNAs into truncated proteins, which impairs tumour-suppressive functions \citep{lee_widespread_2018, singh_widespread_2018}. 

To evaluate the role of APA in mediating genetic effects on gene expression and disease, we sought to identify genetic variants associated with APA on a genome-wide scale. To date, the few studies that have used genome-wide methods to identify variants associated with APA (apaQTLs) have used existing RNA-seq data to infer PAS locations and usage \citep{li_genetic_2019, yoon_genetics_2012, yang_snp2apa_2019, bonder_systematic_2019, mariella_length_2019-1}. While using existing RNA-seq to study APA is economical, identifying PAS and estimating usage using RNA-seq are error-prone and often imprecise \citep{ha_qapa_2018}. Furthermore, using standard RNA-seq data alone to study APA is not informative with regards to whether inter-individual differences in PAS usages are the result of variation in transcriptional termination site choice, or isoform-specific decay or export. Here, we used 3' RNA-seq (3' Seq) to measure PAS usage in steady-state mRNA collected from whole cells as well as mRNA collected from the nucleus, which is comprised of a high proportion of nascent mRNA. This design allowed us to study the effect of genetic variation on isoform PAS at multiple stages of the mRNA lifecycle. Importantly, we collected these data from a panel of human lymphoblastoid cell lines (LCLs) that were previously profiled in great molecular detail, including measurements at the chromatin, RNA, and protein levels \citep{degner_dnase_2012, mcvicker_identification_2013, li_rna_2016, pickrell_understanding_2010}. Integrating the apaQTLs we identified with previously collected molecular data allowed us to study the impact of APA variation on the major steps of the gene regulatory cascade (Figure \ref{fig:3prime4APA}A). We use these data to show that genetic effects on APA can affect virtually all steps of gene regulation (mRNA expression level, translation rate, and protein expression level), and that such effects can impact protein expression, without affecting RNA expression.

\section{Results}\label{ch02-results}

\subsection{Alternative polyadenylation in human LCLs as defined using Nuclear and Total mRNA 3' Seq}\label{APA-LCL-total-nuc}

To measure inter-individual variation in APA, we quantified PAS usage in a panel of 52 Yoruba HapMap LCLs. These same cell lines have been the subjects of multiple studies of gene regulation over the last decade \citep{degner_dnase_2012, mcvicker_identification_2013, li_rna_2016, pickrell_understanding_2010}. We applied 3' Seq to mRNA collected from whole cells (total mRNA fraction) of 52 LCLs and used a peak calling approach (Methods) to comprehensively identify PAS and estimate their usage (Figure \ref{fig:3prime4APA}B,C). Our approach obviates the need for existing annotations, which are biased towards highly expressed isoforms or isoforms expressed in well studied cell-types with higher RNA-seq coverage. In addition, to capture polyadenylated mRNA that may be under-represented or absent in the total mRNA fraction due to rapid turnover, we separately applied 3' Seq to mRNA from isolated nuclei (nuclear fraction) of the same 52 LCLs (Supplementary file 1). Because 3' Seq uses polyA priming to capture the location of polyadenylation sites and is therefore prone to internal priming at transcribed regions that are $A$-rich, we carefully filtered our data to ensure a minimal effect of mispriming on the set of PAS we considered (Method). Specifically, similar to methods previously described, we filtered both individual reads and PAS that map to genomic regions with 70\% A nucleotides or a stretch of 6 A's in the 10 nucleotides upstream \citep{sheppard_accurate_2013, tian_large-scale_2005}. After quality control and filtering, we defined the usage of each PAS in a sample as the ratio of the number of reads that map to the PAS to the number of reads that map to all PAS for the same gene (Figure \ref{fig:3prime4APA}C) (Methods). Thus, we measure the usage of a PAS as the fraction of transcripts using that PAS over the total number of transcripts from the same gene.


We identified 41,810 nuclear PAS in 15,043 genes with at least 5\% mean usage across the 52 LCL samples. We found that 67\% of the protein coding genes expressed in LCLs harbor multiple PAS, suggesting that APA can impact the regulation of most genes \citep{tian_alternative_2017,  mayr_evolution_2016, shi_alternative_2012}. Interestingly, we identified a slight negative correlation between the expression level of a gene and the number of PAS identified for the gene (Pearsons Correlation = -0.12, $p = 2.2 \times10^{-16}$ ). In particular, genes with a single PAS tend to be expressed more highly than genes with multiple PAS. This observation is counter-intuitive from a statistical perspective, and it shows that, in general, out ability to detect PAS was not limited by 3' Seq coverage (Supplementary Figure \ref{fig:ch02-pas-exp}, Methods). We found that the polyA binding protein motif (AATAAA), also known as the polyadenylation signal site, is the most strongly enriched motif in the 50bp regions upstream of our PAS (hypergeometric test, $p < 10^{-391}$).

 We observed that PAS in the 3' UTR are more likely to have a polyadenylation signal compared with intronic PAS ($p < 10^{-16}$, difference of proportion t-test, 75.0\% vs 24.8\%,) (Figure \ref{fig:3prime4APA}D, Supplementary Figure \ref{fig:SS}) and that nearly half (48.3\%) of all 41,810 PAS we identified are located in 3' UTRs (19.4x enrichment) \citep{singh_widespread_2018}. Nevertheless, despite an overall depletion of PAS in introns (0.35x genome-wide levels), we found that the number of PAS in introns is notable (12,793/41,810; 30.6\%) (Figure \ref{fig:3prime4APA}E, Supplementary Figure \ref{fig:totaUsage}).  While signal sites were more highly enriched near 3' UTR PAS than intronic PAS, PAS in introns show clear enrichment of polyadenylation motif 10--50 bp upstream of the cleavage site compared to background intronic sequences (24.8\% vs 0.24\% $p < 10^{-16}$, difference of proportion t-test, Figure \ref{fig:3prime4APA}D).  Thus, the recognition of intronic polyadenylation signals is a general mechanism that can result in premature termination of transcription. 
 
 We  tested the hypothesis that the intronic PAS we identified correspond to truncated mRNA transcripts that escaped telescripting, whereby the U1 snRNP protects introns from premature cleavage and polyadenylation \citep{kaida_u1_2010, berg_u1_2012, oh_u1_2017}. Because the main role of U1 snRNP is to bind and recognize 5' splice sites, the telescripting model predicts that weaker 5' splice sites can result in decreased U1 snRNP affinity for an intron and thus higher rates of early cleavage and polyadenylation. We estimated 5' splice site strength for all intron using MaxEntScore \citep{yeo_maximum_2004} and found that introns with the weakest 5' splice sites harbored more PAS than introns with stronger 5' splice sites (1.5x fold difference, first decile vs remaining deciles, hypergeometric test $p = 8.07\times10^{-87}$, Supplementary Figure \ref{fig:splicesite}, Methods) \citep{tian_widespread_2007}. Moreover, we found that the top 10\% of most highly used intronic PAS have weaker 5' splice sites than introns with lowly used PAS or a random set of introns (Mean MaxEntScore 6.43 vs 7.26 vs 7.48, $p=1.4\times10^{-3}$, Wilcoxon rank sum test). These observations are consistent with the hypothesis that telescripting protects nascent transcripts from early cleavage and polyadenylation in introns, and that the intronic PAS we observe result from transcripts that escape telescripting \citep{kaida_u1_2010, berg_u1_2012, oh_u1_2017}.


We observed that intronic PAS have on average lower usage across individuals than PAS located in 3' UTRs (16.9\% vs 46.2\%). Lower usage of intronic PAS may be explained by weaker polyadenylation signals at intronic PAS compared to 3' UTR PAS or by the impact of telescripting on intronic polyadenylation. However, we hypothesized that some intronic PAS have low usage because premature polyadenylation at intronic sites can produce short-lived transcripts that are rapidly degraded and thus are under-represented in the total mRNA fraction. To test this hypothesis, we identified PAS that are used more often, or exclusively, in the nuclear fraction compared to the total mRNA fraction. By comparing PAS usage estimated in the nuclear and total mRNA fractions from all 52 individuals, we identified at 10\% FDR 591 PAS in 585 genes that are used at least 20\% more in the nuclear compared to the total mRNA fraction. Of these 591 PAS, 134 were found to be used by 1\% or less of the transcripts in the total mRNA fraction, suggesting that these transcripts may be absent from the cytoplasm (Figure \ref{fig:3prime4APA}E, Supplementary Figure \ref{fig:locdPAS}, Methods). Notably, we found that 387 of the nuclear-enriched PAS are intronic (Supplementart Figure \ref{fig:locdPAS}), a large proportion of which (83.4\% vs 43\% for all PAS) are absent from a comprehensive annotation of PAS compiled from 78 human studies that used 3' Seq (Methods, Supplementary Figure  \ref{fig:compAnno}) \citep{wang_polya_db_2018}. While no other study has directly measured PAS usage in nuclei, a proportion of the nuclear enriched intronic sites have been identified in a number of human tissues (up to 10\%, Supplementary file 1). These findings suggest that mRNA transcripts are terminated and polyadenylated in introns at a higher frequency than generally appreciated, and that many of these isoforms escape detection from studies of total mRNA fraction owing to their rapid decay or their propensity to remain within the nucleus. 

\begin{figure}
\centering
\includegraphics[width=5in]{img/ch02/figure1.pdf}
\caption[3' Sequencing of nuclei reliably captures alternative polyadenylation]{\small {\bf (A)} Schematic of how genetic variants affect phenotypes by percolating through gene regulatory layers (black arrows). We aimed to understand how genetic variation can mediate gene regulation through alternative polyadenylation (orange arrows). {\bf (B)} {\it (Left)} Schematic of Lexogen Reverse Quant Seq protocol for 3' Sequencing \citep{moll_quantseq_2014} {\it (Right)}Meta gene plot showing read coverage for five 3' Seq libraries collected from nuclei isolated from LCLs. {\bf (C)} Representation for how PAS usage is calculated. Read count for each PAS were divided by the total number of reads at all PAS for the gene. {\bf (D)} {\it (Main)} Stacked density of canonical (AATAAA, AATTAA) and other polyadenylation signal sites (AAAAAA, AAAAAG, AATACA, AATAGA, AATATA, ACTAAA, AGTAAA, CATAAA, GATAAA, TATAA) upstream of identified PAS.} 
\label{fig:3prime4APA}
\end{figure}


% Continues caption on next page. Requires package ccaption.
\begin{figure}[!htb]
\contcaption{(continued) {\it (Inset)} Proportion of PAS in different genomic regions with a polyadenylation signal site 10-50bp upstream of cleavage site. The red dotted line represents the proportion of signal site in random 40bp windows, i.e. the intronic background. {\bf (D)} The blue line represents the number of PAS identified as the stringency of the usage cutoff increases. The orange and green lines represent the proportion of PAS in the 3' UTR and introns, respectively. The proportion of intronic PAS increases as the usage cutoff decreases, implying that a disproportionate number of intronic PAS are used at low frequencies.}
\end{figure}
  
  


\subsection{Genetic loci associated with variation in APA}\label{apa-QTLs}

Having established that APA can contribute to the generation of complex transcript isoforms, we sought to identify genetic loci associated with inter-individual variation in APA. We normalized each PAS usage ratio using LeafCutter  \citep{li_annotation-free_2018} and tested {\it cis}-associations between genetic variants and PAS usage, correcting for batch and the top principal components (Methods, Supplementary Figure \ref{fig:QQplots},Supplementary Figure \ref{fig:propPASdAPA},Supplementary Figure \ref{fig:PCA}). Using 3' Seq data from the nuclear fraction, we identified 602 nuclear apaQTLs in 479 genes at 10\% FDR. In the total mRNA fraction, we identified 443 apaQTLs in 353 genes at 10\% FDR. For example, individuals with the C/C genotype (rs11032578) show higher usage of an intronic PAS in the {\it ABTB2} gene compared to individuals that are heterozygous C/T or homozygous T/T (Figure \ref{fig:qtlFigure}A). In both fractions, apaQTL lead SNPs are enriched near the PAS they most strongly correlate with and near the 3' ends of gene bodies (Figure \ref{fig:qtlFigure}B, Supplementary Figure \ref{fig:totQTLloc}). The proximity of the apaQTL lead SNPs to PAS may suggest that genetic variants that affect polyadenylation signal motifs drive most of the genetic effects on APA. Although we observed an enrichment of apaQTLs in signal motifs, genetic variants that alter signal motifs are unlikely to explain the majority of apaQTLs (Supplementary Figure \ref{fig:totQTLloc}).

Our study design provides the unique opportunity to evaluate the likely mechanisms by which genetic variation controls PAS usage. While previous studies have demonstrated that genetic variants can impact PAS usage, it has been difficult to discern whether the variation in PAS usage is primarily driven by genetic effects on cleavage and polyadenylation (Figure \ref{fig:qtlFigure}C, Model 1), or on the mRNA lifecycle (e.g. by impacting miRNA binding sites and decay) (Figure \ref{fig:qtlFigure}C, Model 2). We reasoned that if genetic effects functioned primarily by affecting post-transcriptional regulation such as decay or export, then this effect would be detectable in the total mRNA fraction, but would be smaller or undetectable in the nuclear mRNA fraction (Supplementary file 1). Interestingly, we found that only 97 apaQTLs (of 443 apaQTLs, 21.9\%) identified in the total mRNA fraction were not detected in the nuclear mRNA fractions and these associations are much weaker than shared apaQTLs (Supplementary Figure \ref{fig:totSpeWeak}). We thus suspect that we currently lack statistical power to detect most of these 97 apaQTLs in the nuclear mRNA fraction. To estimate sharing of apaQTLs across the two mRNA fractions, we used Storey's $\pi_0$ statistics and found that the vast majority of apaQTLs identified in the total mRNA fractions were estimated to also affect PAS usage in the nuclear mRNA fraction ($\pi_{1}$=0.87, Supplementary Figure \ref{fig:QTLshare}). In addition, we found that the genetic effect sizes on PAS usage were very similar across the two mRNA fractions ($r^{2}$ = 0.66; $p = 10^{-16}$, Figure \ref{fig:qtlFigure}D, Supplementary Figure \ref{fig:apaQTLcorr}). Altogether these observations show that most genetic variants impact PAS usage by affecting polyadenylation site choice. Supporting this notion, we found weak or no enrichment of apaQTLs in sites bound by RNA binding proteins as identified using eCLIP data from ENCODE (Supplementary file 1).

\begin{figure}
\centering \includegraphics[width=5in]{img/ch02/figure2.pdf}
\caption[Identify genetic variation driving differences in polyadenylation as apaQTLs]{\small {\bf (A)} An apaQTL in the {\it ABTB2} gene impact usage of an intronic PAS. {\it (Top)} Gene track and identified PAS. Each bar represents a potential isoform. The red bar corresponds to the isoform most strongly associated with the apaQTL. The vertical dotted line represents the position of the lead apaQTL SNP. {\it (Bottom)} Boxplot of polyadenylation site usage at each PAS by genotype listed according to the isoform order above. The C allele increases usage of the intronic PAS. {\bf (B)} ({\it Left})  Location of the lead nuclear apaQTL SNPs relative to their corresponding PAS. ({\it Right}) Meta gene plot showing the distribution of apaQTL SNPs in the annotated gene body, where 0 represents the transcription start site and 1 represents the annotated transcription end site.} 
\label{fig:qtlFigure}
\end{figure}


\begin{figure}[!htb]
 \contcaption{(continued) {\bf (C)} Two mechanistic models for how genetic variants can affect PAS usage. ({\it Model 1}) Genetic variation acts directly on PAS choice. In this case, the apaQTL will be identified with similar effect sizes in both nuclear and total mRNA fractions, or smaller effect size in the total mRNA fraction.  ({\it Model 2}) Genetic variation acts through a post transcriptional mechanism. For example, one mRNA isoform is subject to decay. In this case, the apaQTL will be identified only in the total mRNA fraction, or will be identified in the total mRNA fraction with a larger effect size than in the nuclear mRNA fraction.{\bf (D)} Effect sizes of apaQTLs originally identified at 10\% FDR in the nuclear mRNA fraction plotted against the effect sizes ascertained in the total mRNA fraction. Regression line is shown in blue and $y = x$ line is shown in black.}
 \end{figure}
 
 
 

\subsection{Impact of apaQTLs on gene expression levels}\label{apa-QTLs-expression}


While we believe that nearly all genetic variants impact PAS usage by affecting polyadenylation site choice and not isoform-specific decay or export, this model is not incompatible with a model in which genetic variants can sometimes impact expression by affecting APA. For example, a genetic variant might increase the relative production of an isoform that is less stable, in which case total transcript levels would decrease. Therefore, next, we asked whether genetic variants could impact gene expression levels by direct effects on APA. We hypothesized that this mode of genetic regulation may be prevalent, in particular for genes with intronic PAS, because isoforms using intronic PAS are often subject to rapid decay. In this model, the genetic effect changes the relative production of isoforms with different relative stabilities rather than specifically modulating the stability of an isoform e.g. by increasing affinity for microRNA binding in the 3' UTR.

To test this hypothesis, we focused on the set of 602 apaQTLs that we identified in the nuclear mRNA fraction, representing genetic variants that impact PAS choice. Our hypothesis predicts that genetic variants that increase intronic PAS usage should decrease gene expression levels. In line with this prediction, we found a negative correlation between the genetic effect sizes for intronic PAS usage and mRNA expression levels ($p = 8.97 \times 10^{-7}$, Figure \ref{fig:apaandexpression}A, Supplementary Figure \ref{fig:3anooutlier}). Thus, our analysis suggests a widespread mechanism whereby genetic variants decrease mRNA expression levels by increasing choice of isoforms with premature PAS that are subject to rapid decay. Of interest, we found that 13 apaQTLs that were detected only in the nuclear fraction are also eQTLs, which highlights the importance of considering early stages of the mRNA lifecycle to uncover eQTL mechanisms.

To further investigate the contribution of APA to gene expression, we sought to understand the relationship between apaQTLs and a set of eQTLs that we previously classified as those with explained putative mechanisms, explained eQTLs (1164 eQTLs, $\sim$ 60\%) or as unexplained eQTLs (801 eQTLs, $\sim$ 40\%) using data from the same LCLs \citep{li_rna_2016}. The eQTLs with explained putative mechanisms were associated with chromatin-level phenotypes including DNase-I hypersensitivity, histone marks, or DNA methylation, and thus are likely to be mechanistically explained by effects mediated by chromatin-level phenotypes (e.g. enhancer or promoter activity). To test whether apaQTLs might account for unexplained eQTLs, we first asked whether genes with unexplained eQTLs were more likely to also harbor apaQTLs than compared to genes with explained eQTLs. Indeed, we found a significantly higher enrichment of low p-value associations with APA for genes with unexplained eQTLs ($p = 0.01$, Figure \ref{fig:apaandexpression}B, Supplementary Figure \ref{fig:totunexp}) and significantly larger absolute apaQTL effect sizes for unexplained eGene compared to explained eGenes (0.35 vs. 0.3, Wilcoxon Rank sum test, $p=6.6 \times 10^{-4}$). We also found that apaQTLs exhibited an association with chromatin states that was more similar to the unexplained eQTLs than the explained eQTLs. In particular, apaQTLs and unexplained eQTLs were more likely to lie in regions of transcription elongation or are associated with weak transcription, and less likely to lie in enhancers or promoters than explained eQTLs (Supplementary Figure \ref{fig:chromHMM}, Methods). Overall, we estimated that 17.3\% of otherwise unexplained eQTLs were associated with PAS usage (see Methods). For example, an unexplained eQTL for \textit{C10orf88} (rs7904973) colocalizes with an apaQTL associated with increased usage of an intronic PAS (Figure \ref{fig:apaandexpression}C). More generally, we found that eQTLs and apaQTLs colocalize for the majority of genes that had both (Methods, Supplementary file 1). These observation thus highlights APA as one important mechanism by which genetic variation impacts gene expression independent from enhancers and promoters. 
  

\begin{figure}
\centering \includegraphics[width=5in]{img/ch02/figure3.pdf}
\caption[apaQTLs provide mechanistic evidence eQTLs ]{\small {\bf (A)} Scatter plot of intronic apaQTL effect sizes plotted against their eQTL effect sizes shows negative correlation. {\bf (B)} Quantile-quantile (Q-Q) plot for apaQTLs shows that apaQTLs are more highly enriched in unexplained eGenes (purple dots) compared to explained eGenes (red dots). {\bf (C)} Example of an apaQTL that is also an unexplained eQTL for {\it C10orf88}. {\it (Top)} Gene track and identified PAS in the {\it C10orf88} gene. The red bar corresponds to the isoform most strongly associated with the apaQTL. The vertical dotted line represents the position of the strongest apaQTL SNP. {\it (Middle)} Zoomed version of track represented above. {\it (Bottom)} Boxplot of polyadenylation site usage at each PAS by genotype listed according to isoform order above. {\bf (D)} {\it (Top)} LocusZoom plot for eQTL associations for the {\it C10orf88} gene. ({\it Bottom)} Locus zoom plot for apaQTL associations. Interestingly, the lead apaQTL and eQTL SNP, rs7904973, has been linked to increased LDL cholesterol through GWAS \cite{klarin_genetics_2018}.}
\label{fig:apaandexpression}
\end{figure}


\subsection{APA mediates gene regulation independently of mRNA expression levels}\label{APA-independent-expression}

Previous joint analyses of molecular QTLs suggested that functional genetic variants tend to affect gene regulation in a simple and straightforward manner: first impacting chromatin activity, then mRNA expression, and finally protein expression \citep{li_rna_2016, battle_genomic_2015}. However, because isoforms with different 3' UTRs have been shown to vary in terms of their translation efficiency, we hypothesized that apaQTLs can impact ribosome occupancy and protein expression levels without affecting mRNA expression levels \citep{floor_tunable_2016-1}. To test this possibility, we asked whether apaQTLs are enriched among genes without a known eQTL, but that are associated with a ribosome occupancy QTL (riboQTL) or a protein expression QTL (pQTL). Indeed, we found that apaQTLs are enriched among genes with a ribosome QTL (rGenes; Wilcoxon rank sum test $p=0.01$) and genes with a pQTL (pGenes; $p = 0.0006$) compared to genes with no molecular association (Figure \ref{fig:expInd}A, Supplementary Figure \ref{fig:totpgene}) \cite{li_rna_2016, battle_genomic_2015}. In addition, we observed a small but significant positive correlation between individual variance in APA usage and ribosome occupancy  (correlation = 0.15, $p <2.2\times10^{-16}$, Supplementary file 1), supporting a model in which APA impacts translation efficiency.

In total, we found 24 apaQTLs that affect protein expression, but not mRNA expression (Table \ref{tab:ch02-s1}). Of these, five apaQTLs were significantly associated with ribosome occupancy (Table \ref{tab:ch02-s1}). This finding is particularly noteworthy because nearly all genetic effects on ribosome occupancy have been proposed to be mediated by effects on mRNA expression \citep{battle_genomic_2015}. Yet, here we provide direct evidence that APA can mediate genetic effects on ribosome occupancy without affecting mRNA expression levels. For example, the apaQTL in the {\it EIF2A} gene that is associated with a switched usage of two 3' UTR PAS, colocalizes with a pQTL and a ribosome occupancy QTL (Figure \ref{fig:expInd}B, Supplementary Figure \ref{fig:locuszoom}), but is not associated with {\it EIF2A} mRNA levels (Figure \ref{fig:expInd}B). Interestingly, the QTL in {\it EIF2A} affects usage of two PAS in the same 3' UTR implying that the protein sequence encoded by the two isoforms are identical. Thus, the regulatory associations uncovered at {\it EIF2A} cannot simply be explained by differences in protein isoform stability. Moreover, while differences in 3' UTR are often assumed to play a regulatory function by influencing decay \citep{mayr_regulation_2017}, mechanisms involving RNA decay cannot be operational in this case because steady-state mRNA expression is unchanged. Instead, differences between the two isoforms may reflect differential binding of factors that impact translation \citep{yamashita_translational_2017}, or differential rates of translation re-initiation at the end of a translation cycle \citep{rogers_ribosome_2017}. 

We identified 19 pQTLs that were associated with APA but not steady-state gene expression or ribosome occupancy levels. Two previous studies also reported the discovery of pQTLs that were not eQTLs \citep{battle_genomic_2015, chick_defining_2016}. In both studies, the authors proposed that some genetic effects on protein expression levels were mediated by changes in the protein sequence or by changes in the expression level of interacting proteins, which would manifest post-translationally. Our finding reveals yet another mode of genetic regulation of protein expression level by APA (e.g. by affecting recruitment of interacting proteins). Thus, these findings provide clear evidence that APA can affect protein expression levels without affecting gene expression levels. Altogether, our findings suggest complex modes of gene regulation independent of mRNA expression driven by variation in APA.


\begin{figure}
\centering
\includegraphics[width=5in]{img/ch02/figure4.pdf}
\caption[apaQTLs explain expression independent rQTLs and pQTLs] {\small {\bf (A)} Quantile-quantile (Q-Q) plot for apaQTLs separated by genes in previously detected rQTLS (red) and pQTLs (purple) that are not eQTLs. Black points are apaQTL genes with no pQTL, rQTL, or eQTL. {\bf (B)}{\it (Top)} Gene track and identified PAS in the {\it EIF2A} gene. The red bar corresponds to the isoform most strongly associated with the apaQTL. The vertical dotted line represents the position of the strongest apaQTL SNP. {\it (Middle)} Zoomed version of track represented above. {\it (Bottom Left)} Boxplot of polyadenylation site usage at each PAS by genotype listed according to isoform order above.  {\it (Top Right)} Boxplot showing normalized mRNA expression for {\it EIF2A} by genotype at the apaQTL SNP (rs9820529).   {\it (Bottom Right)} Boxplot showing normalized protein expression for {\it EIF2A} by genotype at the apaQTL SNP (rs9820529).}
\label{fig:expInd}
\end{figure}


\subsection{APA mediates genetic effects on complex traits}\label{APA-traits}

Genetic variation may impact disease risk through APA. We asked whether common variants in the regions around PAS, i.e. regions enriched for apaQTLs, are enriched in disease heritability. Using LDscore regression to estimate the heritability enrichment of 35 traits in 1kb regions centered around PAS, we found that 14 of the tested traits were significantly enriched (\ref{fig:ldregress}). Of note, genetic variation around PAS was estimated to tag 15.35\% of the SNP heritability for rheumatoid arthritis (7.88 fold enrichment, $p = 0.0025$). We further asked whether we could identify specific apaQTLs associated with phenotype. Indeed, 19.3\% of apaQTLs (including SNPs in LD with $r^{2} > 0.9$) are significantly associated with at least one trait in the UCSC GWAS catalog (Methods) \citep{kent_human_2002}. For example, an apaQTL that colocalizes with the eQTLs in the C10orf88 gene (rs7904973) has been associated with increased LDL cholesterol \citep{klarin_genetics_2018}, suggesting that eQTLs mediated by APA can impact organismal phenotype. Taken together, we propose that APA is a complex regulatory mechanism relevant to our understanding of how genetic variation can affect disease. Thus, comprehensive maps of apaQTLs can enhance our ability to interpret GWAS loci, particularly when the implicated variants are not eQTLs \citep{joehanes_integrated_2017, lee_widespread_2018}. For example, an apaQTL in the {\it ELL2} gene (rs56219066) is correlated with increased usage of an intronic PAS and is associated with risk for multiple myeloma \citep{swaminathan_variants_2015}. Interestingly, multiple myeloma is among the cancer types in which widespread dysregulation of intronic APA has been documented previously \citep{singh_widespread_2018, lee_widespread_2018}.



\section{Discussion}\label{ch02-discussion}

Obtaining a comprehensive understanding of the mechanisms that affect gene regulation is crucial for the functional interpretation of noncoding genetic variation. Yet, existing studies that examine the role of genetic variation on APA are generally characterized by two important shortcomings. Firstly, the study of inter-individual variation in PAS usage have been mostly restricted to APA in the 3' UTRs \citep{li_genetic_2019, yoon_genetics_2012, yang_snp2apa_2019}, leaving genetic variants that impact PAS usage in other regions, e.g. intronic PAS, understudied. Secondly, nearly all existing studies use standard RNA-seq to estimate PAS usage, which not only limits the accuracy of usage quantification, but also makes it difficult to disentangle the contribution of co-transcriptional mechanisms to APA regulation from post-transcriptional mechanisms such as isoform-specific decay. In this study, we overcome these shortcomings by applying 3' Seq to total and nuclear mRNA fractions separately to directly measure PAS usage including that of PAS in intronic regions. 

It is worthwhile to note here that despite the many advantages of using 3' Seq to identify and quantity APA isoforms, 3' Seq experiments are known to be susceptible to mispriming, which occurs when polydT primers designed to recognize the polyA-tail of transcripts anneal to adenosine stretches within a transcript, thus introducing false positive polyadenylation sites. While we used stringent criteria to reduce the effect of mispriming, we found that a small proportion of PAS used in this study may be the result of mispriming. In particular, we found an enrichment of adenosine nucleotides at a subset of intronic PAS which were discovered in our study and not previously annotated, suggesting that 10--20\%{} of unannotated intronic PAS may be false positives (Supplementary file 1). To ensure that these false positive PAS do not affect the validity of our analyses, we performed the main analyses presented in this study after removing unannotated intronic PAS and found that our conclusion were robust to the small number of potential false positive intronic PAS (Supplementary file 1) \citep{wang_polya_db_2018}. 


By collecting data from both total and nuclear mRNA factions, we were able to study the effects of genetic variation on polyadenylation at multiple stages of the mRNA lifecycle, and to distinguish putative regulatory mechanisms by noting the stages at which the genetic effects on APA were observed.  For example, genetic variants can impact steady-state isoform ratio either co-transcriptionally by affecting PAS choice during transcription (Figure \ref{fig:qtlFigure}C top), or  post-transcriptionally by affecting binding of miRNAs or RNA-binding proteins and consequently isoform decay (Figure \ref{fig:qtlFigure}C bottom). We found that the vast majority of genetic variants that affect PAS usage ratio in total mRNA fraction, were also found to have similar effect sizes on PAS usage ratio in the nucleus. This observation implies that inter-individual variation in steady-state APA levels can generally be explained by variation in co-transcriptional mRNA processing, or mRNA processing that occur soon after transcription. 

There are several co-transcriptional mechanisms that may result in variation in PAS usage. For example, previous reports have suggested that variation in the polyadenylation signal site may cause variation in PAS usage. While we found that this was the case for a small number of examples, disruption of canonical signal motifs does not appear to be a major mechanism for generating apaQTLs, an observation that is also supported by a recent study on APA in GTEx data (Supplementary Figure \ref{fig:ssDisrupt}) \citep{li_genetic_2019}. Other possible co-transcriptional mechanisms involved in PAS choice include competition between the spliceosome and polyadenylation factors for example mediated by the spliceosomal RNA U1 \citep{oh_u1_2017}, and RNAP II pausing \citep{fusby_coordination_2016}. Indeed, recent studies have reported that sequence and chromatin context can pause or slow down RNAP II elongation across the gene body \citep{mayer_native_2015}, suggesting that variation in RNAPII pausing may impact PAS choice \citep{fusby_coordination_2016}. For example, in Drosophila melanogaster, paused RNAPII promotes the recruitment of ELAV on the pre-mRNA, which prevents usage of a proximal PAS \citep{oktaba_elav_2015}. In addition, Liu et al. observed a tissue-specific shift toward usage of proximal PAS sites in Drosophila melanogaster mutant for a slow elongation form of RNAPII \citep{liu_transcription_2017}. These findings further suggest that variants affecting RNAPII elongation rate could underlie the genetic effects on PAS usage we detected in this study. 

Although our data suggest that apaQTLs do not generally impact rates of mRNA decay, e.g. by affecting miRNA or RBP binding motifs, we found clear evidence that apaQTLs may promote polyadenylation site choices that result in the production of isoforms with different rates of decay. For example, we observed that genetic variants that increase the usage of isoforms ending at intronic PAS tend to be associated with lower levels of gene expression. This observation is consistent with reports that isoforms with premature polyadenylation are often substrates for nonsense mediated decay or nonstop decay \citep{tian_alternative_2017, vasudevan_non-stop_2002}. More generally, our results suggest that apaQTLs can affect gene expression levels post-transcriptionally by impacting the production of isoforms with varying levels of stability. Importantly, our study highlights APA as an eQTL mechanism independent of promoters and enhancers. 


While the effect of genetic variants on gene regulation is generally assumed to move linearly from chromatin, to mRNA, to protein level, our study reveals several complex modes of genetic regulation for both gene expression and protein expression levels by APA. Although we were unable to study the genome-wide effects of APA on protein expression owing to a scarcity of protein-level data, we identified several apaQTLs that affect protein, but not gene expression levels. These results strongly suggest that APA can affect protein expression levels without affecting gene expression levels, because our power to detect genetic effects on gene expression levels far exceeds that to detect genetic effects on protein expression levels. Furthermore, some of these pQTLs were associated with ribosomal occupancy and some were not, which implies multiple pathways by which genetic variants can impact protein expression levels through APA.

In conclusion, there are many pathways through which genetic variants can impact gene regulation and, consequently, organismal phenotypes. While many studies have demonstrated the importance of gene expression regulation through promoters or enhancers, very few studies have focused on co- or post-transcriptional gene regulation. Our study shows that co- and post-transcriptional processes such as APA can mediate the effects of a substantial number of genetic variants on mRNA expression levels, protein expression levels, and risk for complex diseases.


\section{Methods}\label{ch02-methods}

\subsection{Cell Culture}\label{ch02-cell-culture}

We cultured 54 Epstein-Barr virus transformed LCLs under identical conditions at 37 C and 5\% CO2. These LCLs were derived from Yoruba individuals originally collected as part of the HapMap project \cite{international_hapmap_consortium_haplotype_2005}.The sampleIDs and Research Resource Identifiers (RRIDs) can be found in online version of paper (see chapter citation). Details for each cell line are found in Supplementary file 3. We grew cells in a glutamine depleted RPMI [RPMI 1640 1X from Corning (15-040-CM)], completed with 15\% FBS, 2mM GlutaMAX (from gibco (35050-061), 100 IU/ml Penicillin, and 100 ug/ml Streptomycin. After passaging them 3 times the lines were maintained at a concentration of $1\times10^{6}$ cells per mL. In preparation for extraction, we allowed the cells to grow until a concentration of $1\times10^{6}$ cells per mL was reached and then proceeded to extraction. \

\subsection{Collection and RNA extraction}\label{Collection-and-RNA-extraction}

We collected 30 million cells from each line and divided them into two 15 million cell aliquots. We spun the cells down at 500 RPM at 4C for 2 min, and then washed the pellets with phosphate-buffered saline (PBS) and spun down again. After this we aspirated the PBS, leaving the cell pellet. All washing steps occurred on ice or in cooled centrifuges. At this point every cell line had two separate pellets each from an input of 15 million cells. From each line we took one of these pellets for nuclear isolation. We then carried out nuclear isolation using the nuclear isolation steps outlined by \citep{mayer_genome-wide_2016}. Once we washed and spun down the pellets in the nuclei wash buffer, we resuspended them in 700 ul of the QIAzol lysis reagent (Qiagen). We extracted both RNA cell pellets from the same line in the same batch using the miRNeasy kit (Qiagen) according to manufacture instructions, including the DNase step to remove potentially contaminated genomic DNA. Details for the collection such as cell viability and cell concentration at time of collection are found in Table \ref{tab:ch02-s2}. We checked the quality of the collected RNA using a nanodrop. RNA concentrations and absorbance levels from the collection are in Table \ref{tab:ch02-s2}.

In order to verify fraction separation, we completed the Mayer and Churchman protocol to isolate chromatin and collected cell lysates for each step in the fractionation \citep{mayer_genome-wide_2016}. We performed western blots against both GAPDH (GAPDH antibody (6C5) Life Technologies AM4300) and the Carboxyl Terminal Domain of Pol-II (CTD) (Pol II CTD Ser5-P antibody, Active Motif, 61085). We ran each lysate on Mini-protean TGX precast gels (bioRad 456-1093) after digesting any remaining DNA molecules from the nuclear isolate with benzonase nuclease. We used Goat anti-Mouse IgG (H+L) (Invitrogen 32430) as a secondary antibody for the GAPDH antibody and Goat anti-Rat IgG (H +L) (Invitrogen 31470) as a secondary antibody for the CTD antibody. We diluted all antibodies in a 1:1000 dilution with blocking solution made from dry milk (LabScientific Lot 1267N Cat M0841). We show GAPDH isolated in the cytoplasm and CTD to the chromatin fraction (Supplementary Figure \ref{fig:western}).


\subsection{3' Sequencing library generation}\label{three-Sequencing-library-generation}

We generated 108 single-end RNA 3'sequencing libraries from the total and nuclear RNA extract using the QuantSeq 3 ' mRNA-Seq Library Prep Kit \citep{moll_quantseq_2014} as directed by the manufacturer. We used 5ng of each sample as input. We submitted the libraries for sequencing on the Illumina NextSeq5000 at the University of Chicago Genomics Core facility using single end 50bp sequencing. 

\subsection{3' Sequencing data processing}\label{three-Sequencing data processing}

We mapped 3'  Seq reads to hg19 \citep{church_modernizing_2011} using STAR RNA-seq aligner \citep{dobin_star_2013} using default settings with the WASP mode to filter out reads mapping with allelic bias \citep{van_de_geijn_wasp_2015}. Similar to previously published 3'  Seq methods, we accounted for internal priming by filtering reads preceded by 6 Ts in a row or 7 of 10 Ts in the 10 bases directly upstream of the mapping position in the reference genome \citep{tian_large-scale_2005, sheppard_accurate_2013, beaudoing_patterns_2000}. We verified the individual identity of all bam files using VerifyBamID \citep{jun_detecting_2012}. Due to low confidence in the identity of 2 individuals, they were removed from all analysis. Raw read and mapped read statistics after accounting for internal priming can be found in Table \ref{tab:ch02-s2} ( Supplementary Figure \ref{fig:nucMap},Supplementary Figure \ref{fig:TotalMap},Supplementary Figure \ref{fig:NucMapCount},Supplementary Figure \ref{fig:TotMapCount}).

\subsection{Identification and characerization of PAS}\label{Identification-and-characerization-of-PAS}

We merged all mapped reads and called peaks using an inclusive method, identifying all regions of the genome with non-zero read counts in 90\% percent of libraries and an average read count of greater than 2 counts. This resulted in 138,181 peaks. We assigned each of these peaks to a genic location according to NCBI Refseq annotations for 5' UTRS, 3' UTRs, exons, introns, and regions 5kb downstream of annotated genes downloaded from the UCSC table browser \citep{kent_human_2002}. When a region mapped to multiple genes we used a hierarchical model, similar to the method used by Lin et al. \citep{lin_-depth_2012} to assign the peak to a gene annotations. Our method prioritizes annotations in the following order: 3' UTRs, 5kb downstream of genes, exons, 5'  UTRs, and introns. To further verify absence of PAS detected as a result of internal priming we removed PAS with 6A's or 70\% As in the 15 basepairs downstream of the site. We next utilized a gene level noise filter to account for non-uniform read coverage across the genome. We created a usage score for each PAS based on of the number of reads mapping to the PAS over the number of reads mapping to any PAS associated with the same gene. We filtered out peaks with a mean usage of less than 5\% in both the total and nuclear libraries. After this filter, we were left with 35,032 PAS in the total mRNA fraction and 39,164 PAS in the nuclear fraction. The merged set with PAS from both fractions used for PAS QC is available on GEO and has 41,810 PAS. We compared our set of PAS to the human PolyADB release 3.2 annotation \citep{wang_polya_db_2018}(Supplementary Figure  \ref{fig:compAnno}). We explored the relationship between number of PAS detected and gene expression using TPM estimates from YRI LCLs after removing very lowly expressed genes (less than 1 TPM) \citep{lappalainen_transcriptome_2013}. We calculated the 5' splice site strength using the MaxEntScore tool, for each of the introns in our annotation \citep{yeo_maximum_2004}. We binned the introns by decile according to the scores and evaluated the distribution of the introns containing PAS. We also used the scores for the introns containing PAS to investigate the relationship between PAS usage and 5' splice site strength.

\subsection{PAS Signal site enrichment and locations}\label{PAS-Signal-site-enrichment-and-locations}

We used the Homer findMotifsGenome.pl script  with the -size -300,100  option to identify binding motifs in the 50bp upstream of each PAS \citep{heinz_simple_2010}. As a background, we used genome shuffle to randomly chose the same number of 50bp regions. To explore the location of the signal site relative to the PAS (most 3' end of each identified peak), we determined the relative position of previously described potential signal sites to this position \citep{beaudoing_patterns_2000}. We then extended each PAS 100bp upstream and identified the starting position of each of the 12 PAS signal site variations identified by Beaudoing et al. without allowing for sequence mismatch  \citep{beaudoing_patterns_2000}. 


\subsection{Differential Isoform analysis}\label{Differential-Isoform-analysis}

We mapped 3' Seq reads to all PAS peaks with mean coverage of 5\% in the total or nuclear fraction libraries. This results in 41,813 annotated sites. We assigned reads to PAS using the featureCounts tool with the -O flag to assign reads to all overlapping features \citep{liao_featurecounts_2014}. We ran the leafcutter\_ds.R script on chromosomes 1-22 separately using the cellular fraction label as the sample group identifier \citep{li_annotation-free_2018}.  This analysis tests 9790 genes and resulted in 8227 genes with significant (FDR 10\%) isoform level differences between the total and nuclear cellular fraction. We called differentially used PAS as sites with a $\Delta$ polyadenylation site usage ($\Delta$ PAU) greater than 0.2 or less than -0.2. In our analysis a positive $\Delta$PAU corresponds to increase usage in the total cellular fraction while a negative $\Delta$ PAU corresponds to increased usage in the nuclear fraction. 

\subsection{apaQTL calling in both fractions}\label{apaQTL-calling-in-both-fractions}

We used the leafcutter prepare\_phenotype\_table.py script with default settings to normalize the PAS usage ratios across individuals within each fraction. This method also outputs the top principal components (PCs) of the data to use as covariates. We plotted the proportion of variation explained by each PC in order to identify the number of PCs to include in the analysis (Figure 2.13). We included the top 4 PCs as well as the library preparation batch as the covariates. We plotted the proportion of variance explained by a number of cofactors in each of the top 10 PCs. (Supplementary Figure \ref{fig:PCA}) The top four PCs correlate most strongly with the cell count at collection (Supplementary Figure  \ref{fig:PCA}). We used the same genotypes from Li et al. 2016\citep{li_rna_2016}, available at http://eqtl.uchicago.edu/jointLCL/genotypesYRI.gen.txt.gz \citep{li_rna_2016}. We removed individual NA19092 due to lack of genotype information in this file, bringing our sample size to 51 individuals for this part of the analysis. Only SNPs with a MAF $>$ 5\% in our sample were included. We used FastQTL to map apaQTLs in cis (25kb on either side) with 1000 permutations to select the top SNP-PAS association \citep{ongen_fast_2016}. We called apaQTLs in each fraction as variants passing 10\% FDR (Benjamini-Hockberg) after permutations. In order to plot interpretable effect sizes for each association we computed nominal PAS:SNP associations for the pre-normalized PAS ratios.


\subsection{Association of apaQTLs with chromatin states}\label{Association-of-apaQTLs-with-chromatin-state}

We downloaded the GM12878 chromatin HMM annotations for Hg19 from the UCSC table browser \citep{kent_human_2002}. We overlapped the eQTLs identified and published in Li et al. 2016\citep{li_rna_2016} as well as the total and nuclear fraction apaQTLs with these categories. We calculated 95\% confidence intervals for each measurement by sampling the number of QTLs in the set with replacement 1000 times (Supplementary Figure  \ref{fig:chromHMM}). 


\subsection{apaQTL overlap with eQTLs}\label{apaQTL-overlap-with-eQTLs}
We obtained the set of explained and unexplained eQTLs from Li et al. 2016 \citep{li_rna_2016}. In order to test whether genes with an unexplained eQTL are more likely to be explained by variation in APA, we separated the permuted apaQTL association (top snp per PAS) into three categories: unexplained eGene, explained eGene, non eGenes. We tested for significant enrichment of apaQTLs in each category using one-sided Wilcoxon rank sum tests. In order to test if each explained and unexplained eQTLs described in Li et al. 2016\citep{li_rna_2016} overlaps with an apaQTL, we extracted the nominal associations for each eQTL gene-SNP pair from the apaQTL data in both fractions. In order to account for multiple PAS associations for each pair, we selected the most significant p-value and used a Bonferroni correction to account for the number of PAS tested in the gene. We consider an eQTL as explained by an apaQTL if the corrected p-value is less than 0.05 but report the values for a range of cutoffs in Supplementary Figure \ref{fig:popExp}.We performed colocalization with the R coloc package \citep{wallace_statistical_2012}. The Bayes Factor colocalization method reports Bayes Factors for 4 alternative hypotheses. PP0: No association with either trait, PP1: No association with trait 1, PP2: No association with trait 2, PP3: Association with trait 1 and trait 2, two independent SNPs, and PP4: Association with trait 1 and trait 2, one shared SNP. If causal SNPs for an apaQTL and an eQTL is the same SNP, then PP4 is expected to be large (greater than 0.5). We accounted for incomplete power using the method described in Ongen et al. (Supplementary file 1) \citep{ongen_estimating_2017}.


\subsection{apaQTLs overlap with ribosome specific and protein specific QTLs}\label{apaQTLs-ribosome-specific-protein-specific-QTLs}
The list of protein specific QTL genes can be found in the supplementary information from Battle et al. 2015\citep{battle_genomic_2015}. In order to show that genes with an eQTL and protein specific QTLs are likely to be associated with APA, we separated the permuted apaQTL association (top snp per PAS) into three categories: eGene, pGene, or neither pGene nor eGene. We performed the same analysis with rGenes, eGenes, and neither rGenes nor eGenes. We tested for significant enrichment with one sided Wilcoxon rank sum tests (Figure \ref{fig:expInd}A, Supplementary Figure \ref{fig:totpgene}). 


\subsection{Identification of molecular QTL associations}\label{Identification-molecular-QTL-associations}
We sought to test if SNPs identified as apaQTLs are significantly associated with other molecular phenotypes previously tested in the same panel of LCLs. We tested for associations between the genotypes used in this study and each gene for each phenotype with fastqtl using the top 5 PCs calculated in Li et al. 2016 as covariates \citep{li_rna_2016}. We used normalized RNA expression, RiboSeq values, and protein levels, published in Li et al. 2016 \citep{li_rna_2016}. 


\subsection{PAS heritability estimates and apaQTL overlap with GWAS Catalog}\label{PAS-heritability-estimates-GWAS-Catalog}
\subsection*{PAS heritability estimates and apaQTL overlap with GWAS Catalog}
We downloaded GWAS summary statistics from both Astle et al. and Okada et al. \citep{astle_allelic_2016, okada_genetics_2014} We augmented our PAS sites by 500bp on either side and ran LD score regression using methods described in Bulik-Sullivan et al. \citep{bulik-sullivan_ld_2015} We downloaded the CRCh37hg19 GWAS catalog for UCSC table browser \citep{kent_human_2002}. We identified SNPs in LD with the nuclear apaQTLs using the LDproxy tool from LDlink with YRI as the population \citep{machiela_ldlink_2015}. We filtered all results to SNPs with an r2 greater than 0.9. We overlapped the full set with the GWAS catalog using pybedtools. 



\subsection{Data and code
availability}\label{ch02-data-and-code-availability}

Fastq files and PAS annotations are available at GEO under accession GSE138197 \url{https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE138197}. All reproducible scripts and software versions can be found at \url{https://brimittleman.github.io/apaQTL/} A versioned release of the github is available through Zenodo with doi:10.5281/zenodo.3905372 \url{https://zenodo.org/record/3905372#.XvKD4S2ZMXp}

\section{Acknowledgments}\label{ch02-acknowledgments}

We thank N. Gonzalez, J.P. Staley, M.C. Ward for comments on the manuscript. \textbf{Funding}: This work was supported by the US National Institutes of Health (R01GM130738 to  Y.I.L). B.E.M. supported by T32 GM09197 to the University of Chicago and F31HL149259 to B.E.M. from National Heart, Lung, And Blood Institute of the National Institutes of Health. SP was in part supported by the National Center for Advancing Translational Sciences of the NIH (K12 HL119995). This work was completed in part with resources provided by the University of Chicago Research Computing Center.

\section{Author Contributions}\label{ch02-author-contributions}

 Y.I.L. conceived of the project. B.E.M, S.W. and S.P. performed the experiments. B.E.M analyzed the data with help from Y.I.L, S.P., T.Z., Z.M. and M.K. B.E.M. drafted the manuscript with input from Y.G., Y.I.L, and S.P. S.P., Y.G. and Y.I.L. supervised this project. 

\clearpage
\section{Supplementary Information}\label{ch02-supplementary-information}

\subsection{Supplementary Figures}\label{ch02-supplementary-figures}
\clearpage

\begin{figure}[!htb]
\centering
\includegraphics[width=5in]{img/ch02/Fig1_figuresupplement1.pdf}
\caption[Relationship between Number of PAS and gene expression]{\textbf{Relationship between Number of PAS and gene expression} Relationship between number of PAS identified in our study and gene expression levels (TPM) as measured from GEUVADIS YRI LCLs \citep{lappalainen_transcriptome_2013}. Genes with mean TPM $<$ 1 across individuals were considered not expressed and thus were removed for this analysis.}
\label{fig:ch02-pas-exp}
\end{figure}
\clearpage

\begin{figure}[!htb]
\centering \includegraphics[width=5in]{img/ch02/Fig1_figuresupplement2.pdf}
\caption[Distribution of signal sites upstream of PAS. Supplement to Figure 2.1D]{\textbf{Distribution of signal sites upstream of PAS. Supplement to Figure 2.1D} {\bf (A)} Stacked density plot showing the signal site distribution for PAS in 3' UTR. Other signal sites are AAAAAA, AAAAAG, AATACA, AATAGA, AATATA, ACTAAA, AGTAAA, CATAAA, GATAAA, TATAAA. {\bf (B)} Stacked density plot showing the signal site distribution for PAS in introns. Other signal sites are AAAAAA, AAAAAG, AATACA, AATAGA, AATATA, ACTAAA, AGTAAA, CATAAA, GATAAA, TATAAA.}
\label{fig:SS}
\end{figure}
\clearpage

\begin{figure}[!htb]
\centering
\includegraphics[width=5in]{img/ch02/Fig1_figuresupplement3.pdf}
\caption[Proportion of PAS in 3' UTRs and introns as predicted from total 3' Seq. Additional figures corresponding to Figure 2.1E.]{\textbf{Proportion of PAS in 3' UTRs and introns as predicted from total 3' Seq. Additional figures corresponding to Figure 2.1E.} Number of PAS identified with usage larger than the usage cutoff (x axis) in the total mRNA fraction (purple). Proportion of PAS in introns when PAS are filtered by total usage (green). Proportion of PAS in 3' UTRs when PAS are filtered by total usage (orange).}
\label{fig:totaUsage}
\end{figure}
\clearpage

\begin{figure}[!htb]
\centering \includegraphics[width=5in]{img/ch02/Fig1_figuresupplement4.pdf}
\caption[Intronic PAS 5' Splice site strength]{\textbf{Intronic PAS 5' Splice site strength} Intronic PAS are enriched in introns with the weakest 5' splice sites. Splice site strengths for all introns were calculated using MaxEntScore \citep{yeo_maximum_2004}}.
\label{fig:splicesite}
\end{figure}
\clearpage

\begin{figure}[!htb]
\centering \includegraphics[width=5in]{img/ch02/Fig1_figuresupplement5.pdf}
\caption[Location of PAS differentially used]{\textbf{Location of PAS differentially used} We adapted LeafCutter to identify genes with significant differential usage of PAS between the total and nuclear fraction. The majority of PAS preferentially used in the nuclear fraction are intronic, whereas the majority of PAS preferentially used in the total fraction lie in the 3' UTR. }
\label{fig:locdPAS}
\end{figure}
\clearpage


\begin{figure}[!htb]
\centering \includegraphics[width=5in]{img/ch02/Fig1_figuresupplement6.pdf}
\caption[Comparison of our 3'-Seq PAS to previous PAS annotations]{\textbf{Comparison of our 3' Seq PAS to previous PAS annotations} {\bf (A)} Distribution of distance between PAS and closest annotated site in the annotation database (PolyA\_DB release 3.2) \citep{wang_polya_db_2018}. {\bf (B)} Scatter plot showing the number of PAS we identified in our study (X axis) versus the number of PAS in the PolyA database (Y axis) separated by genomic location (colors). {\bf (C)} Scatter plot showing the number of nuclear-specific PAS we identified in our study versus the number of PAS in the PolyA database separated by genomic location (colors). The vast majority of nuclear-specific PAS are intronic. {\bf (D)} Proportion of PAS present in the PolyA database by usage in nuclear (green) or total (orange) mRNA fraction.}
\label{fig:compAnno}
\end{figure}
\clearpage


\begin{figure}[!htb]
\centering \includegraphics[width=5in]{img/ch02/Fig2_figuresupplement1.pdf}
\caption[Q-Q plots for apaQTLs]{\textbf{Q-Q plots for apaQTLs} {\bf (A)} Q-Q plot for nuclear apaQTLs, plotting adjusted p-values of the top SNP PAS associations. {\bf (B)} Q-Q plot for total apaQTLs, plotting adjusted p-values of the top SNP PAS associations.}
\label{fig:QQplots}
\end{figure}
\clearpage

\begin{figure}[!htb]
\centering
\includegraphics[width=5in]{img/ch02/Fig2_figuresuplement2.pdf}
\caption[Proportion of PAS tested with an apaQTL]{\textbf{Proportion of PAS tested with an apaQTL} Proportion of PAS in different genomic locations with a significant apaQTL. The numbers above each bar represent the number of identified apaQTL for each location. }
\label{fig:propPASdAPA}
\end{figure}
\clearpage

\begin{figure}[!htb]
\centering
\includegraphics[width=5in]{img/ch02/Fig2_figuresupplement3.pdf}
\caption[Analysis of the PCs of APA usage]{\textbf{Analysis of the PCs of APA usage} {\bf (A)} Proportion of variance explained in the 10 first PCs by experimental variables in nuclear APA usage. We used a linear model to look at correlation between PC and each covariate. {\bf (B)} Proportion of variance explained in the 10 first PCs by experimental variables in total APA usage. We used a linear model to look at correlation between PC and each covariate. {\bf (C)} Proportion of variance explained by each PC in APA usage. Vertical line represents the number of PCs used as covariates in our apaQTL analysis.}
\label{fig:PCA}
\end{figure}
\clearpage

\begin{figure}[!htb]
\centering
\includegraphics[width=5in]{img/ch02/Fig2_figuresupplement4.pdf}
\caption[apaQTLs in both fractions are associated with PAS near SNP and at the transcription end site.  Supplement to Figure 2.2B and 2.2C.]{\textbf{apaQTLs in both fractions are associated with PAS near SNP and at the transcription end site.  Supplement to Figure 2.2B and 2.2C.} {\bf (A)} Histogram showing the distribution of the distance between lead apaQTL SNP and the PAS, separated by mRNA fraction. {\bf (B)} Histogram showing the distribution of the distance between lead apaQTL SNP and gene features, where 0 represents annotated TSS and 1 represents annotated TES, separated by mRNA fraction.}
\label{fig:totQTLloc}
\end{figure}
\clearpage

\begin{figure}[!htb]
\centering
\includegraphics[width=5in]{img/ch02/Fig2_figuresupplement5.pdf}
\caption[Signal site disruption]{\textbf{Signal site disruption} {\bf (A)} Q-Q plot showing the nuclear apaQTL p-values for SNP in signal sites upstream of 3' UTR PAS compared to matched SNPs (equal distance) upstream of a set of 3' UTR PAS without identified signal sites. {\bf (B)} Similar to panel A, but for total apaQTLs.}
\label{fig:ssDisrupt}
\end{figure}
\clearpage

\begin{figure}[!htb]
\centering
\includegraphics[width=5in]{img/ch02/Fig2_figuresupplement6.pdf}
\caption[Total mRNA specific apaQTLs show weaker association than do shared apaQTLs]{\textbf{Total mRNA specific apaQTLs show weaker association than do shared apaQTLs} {\bf (A)} Boxplot showing the -log10(p-value) of the nominal total apaQTL associations separated by whether the association is also identified in the nuclear mRNA fraction. ApaQTLs that are total-specific have significantly weaker associations. {\bf (B)} Scatter plot showing the relationship between the -log10(p-valued) of the apaQTL associations in both mRNA fractions for total mRNA apaQTLs. Dots and densities are colored by whether the apaQTL is total-specific or shared. Total-specific apaQTLs are likely not detected in the nuclear fraction due to a lack of power.}
\label{fig:totSpeWeak}
\end{figure}
\clearpage

\begin{figure}[!htb]
\centering
\includegraphics[width=5in]{img/ch02/Fig2_figuresupplement7.pdf}
\caption[apaQTL sharing between fractions]{\textbf{apaQTL sharing between fractions} {\bf (A)}	 Histogram showing the P-value distribution of the apaQTL associations between the lead total apaQTL SNP and the corresponding PAS ascertained using our 3'-Seq data from the nuclear mRNA fraction. Values were calculated based on PAS tested in both fractions (403 of 443). Results are robust to using all PAS $(pi_1=0.842)$ {\bf (B)} Histogram showing the P-value distribution of the apaQTL associations between the lead nuclear apaQTL SNP and the corresponding PAS ascertained using our 3'-Seq data from the total mRNA fraction. Values calculated based on PAS tested in both fractions. (483 of 602) Results are robust to using all PAS $(pi_1=0.825)$}
\label{fig:QTLshare}
\end{figure}
\clearpage


\begin{figure}[!htb]
\centering
\includegraphics[width=5in]{img/ch02/Fig2_figuresupplement8.pdf}
\caption[Correlation of effect sizes for apaQTLs discovered in total and nuclear mRNA fractions]{\textbf{Correlation of effect sizes for apaQTLs discovered in total and nuclear mRNA fractions} {\bf (A)} Normalized effect sizes ascertained in total mRNA and nuclear fraction of total apaQTLs tested in both fractions. {\bf (B)} Normalized effect sizes ascertained in total mRNA and nuclear fraction for nuclear apaQTLs tested in both fractions.}
\label{fig:apaQTLcorr}
\end{figure}
\clearpage

\begin{figure}[!htb]
\centering
\includegraphics[width=5in]{img/ch02/Fig3_figuresupplement1.pdf}
\caption[Figure 2.3A without outlier SNP]{\textbf{Figure 2.3A without outlier SNP} Scatter plot showing the relationship between intronic nuclear apaQTL effect size and eQTL effect size after removing outlier SNPs (Filtered for SNPs with eQTL effect size $<$ -2.0).}
\label{fig:3anooutlier}
\end{figure}
\clearpage

\begin{figure}[!htb]
\centering
\includegraphics[width=5in]{img/ch02/Fig3_figuresupplement2.pdf}
\caption[Overlap between apaQTLs in total fraction and eQTLs, supplement to Figure 2.3B]{\textbf{Overlap between apaQTLs in total fraction and eQTLs, supplement to Figure 2.3B} QQ-plot showing the total apaQTL (adjusted) p-values separated by whether the gene harbors an explained (red) or unexplained (blue) eQTLs. We observe an enrichment for low apaQTL association p-values in genes with eQTLs compared to all tested genes (black). }
\label{fig:totunexp}
\end{figure}
\clearpage

\begin{figure}[!htb]
\centering
\includegraphics[width=5in]{img/ch02/Fig3_figuresupplement3.pdf}
\caption[Proportion of apaQTLs and eQTLs by Chromatin state]{\textbf{Proportion of apaQTLs and eQTLs by Chromatin state} Bar plot showing the proportion of apaQTLs located in each of the 12 chromatin states from chromHMM. We find that the location profile of apaQTLs is more similar to that of unexplained eQTLs than that of explained eQTLs. Error bars represent the 95\% confidence interval for each point estimate from bootstrapping 1,000 times.}
\label{fig:chromHMM}
\end{figure}
\clearpage

\begin{figure}[!htb]
\centering
\includegraphics[width=5in]{img/ch02/Fig4_figuresupplement1.pdf}
\caption[Overlap between apaQTLs in total fraction and eQTLs, rQTLs and pQTLs supplement to Figure 2.4A]{\textbf{Overlap between apaQTLs in total fraction and eQTLs, rQTLs and pQTLs supplement to Figure 2.4A} {\bf (A)} QQ-plot showing the total apaQTL (adjusted) p-values separated by whether the corresponding gene has a ribosome occupancy QTL (red) or an eQTL (red). We see an enrichment for low apaQTL p-values in genes with either association. {\bf (B)} QQ-plot showing the total apaQTL (adjusted) p-values separated by whether the corresponding gene has a protein expression QTL (red) or an eQTL (red). We see an enrichment for low apaQTL p-values in genes with either association.}
\label{fig:totpgene}
\end{figure}
\clearpage

\begin{figure}[!htb]
\centering
\includegraphics[width=5in]{img/ch02/Fig4_figuresupplement2.pdf}
\caption[LocusZoom plots for EIF2A molecular associations, Supplement to Figure 2.4B]{\textbf{LocusZoom plots for EIF2A molecular associations, Supplement to Figure 2.4B} LocusZoom plots for EIF2A apaQTL in Figure 4b along with associations with RNA expression, ribosome occupancy (ribo-seq), and protein expression as determined using normalized data from Li et al. 2016 \citep{li_rna_2016}. LD patterns were colored according to the HapMap YRI lines.}
\label{fig:locuszoom}
\end{figure}
\clearpage



\begin{figure}[!htb]
\centering
\includegraphics[width=5in]{img/ch02/Fig4_figuresupplement3.pdf}
\caption[LD Score regression enrichment estimates suggest that APA regulation is likely relevant for complex human phenotypes]{\textbf{LD Score regression enrichment estimates suggest that APA regulation is likely relevant for complex human phenotypes} {\bf (A)} Percent of heritability explained by SNPs within 1kbp around each PAS.}
\end{figure}


\begin{figure}[!htb]
 \contcaption{(continued) Error bars represent +/- 1 standard deviation.Blood phenotype statistics published in Astle et el \citep{astle_allelic_2016}. Rheumatoid arthritis statistics were obtained from Okada et al \citep{okada_genetics_2014}. {\bf (B)} Enrichment of heritability explained by SNPs within 1kbp around PAS for the phenotypes analyzed.}
\label{fig:ldregress}
\end{figure}
\clearpage

\begin{figure}[!htb]
\centering
\includegraphics[width=5in]{img/ch02/Fig1_figuresupplement7.pdf}
\caption[Western Blots to demonstrate cell fractionation]{\textbf{Western Blots to demonstrate cell fractionation} {\bf (A)} Western blot against Carboxyl terminal domain of RNA Polymerase II, photo captured at 10 second exposure. Blot is not used for quantification, but to validate cell fractionation. {\bf (B)} Western blot against GAPDH to mark glycolysis in cytoplasm, photo captured at 25 second exposure time. Blot is not used for quantification, but to validate cell fractionation. Figure panels are modeled off Mayer and Churchman 2016, Figure 2 \citep{mayer_genome-wide_2016}}
\label{fig:western}
\end{figure}
\clearpage

\begin{figure}[!htb]
\centering
\includegraphics[width=5in]{img/ch02/Fig1_figuresupplement8.pdf}
\caption[3'-Seq read mapping proportions for the nuclear mRNA fraction]{\textbf{3'-Seq read mapping proportions for the nuclear mRNA fraction} Proportion of reads that map to the genome (mapped) and the proportion of final reads used for analysis are cleanly mapped (Clean Mapped) by nuclear mRNA library. Cleanly mapped reads are reads that mapped successfully and passed the filtering for mispriming (MP) as described in the Methods.}
\label{fig:nucMap}
\end{figure}
\clearpage

\begin{figure}[!htb]
\centering
\includegraphics[width=5in]{img/ch02/Fig1_figuresupplement9.pdf}
\caption[3'-Seq read mapping proportions for the total mRNA fraction]{\textbf{3'-Seq read mapping proportions for the total mRNA fraction} Proportion of reads that map to the genome (mapped) and the proportion of final reads used for analysis that are cleanly mapped (Clean Mapped) by total mRNA library. Cleanly mapped reads are reads that mapped successfully and passed the filtering for mispriming (MP) as described in the Methods.}
\label{fig:TotalMap}
\end{figure}
\clearpage

\begin{figure}[!htb]
\centering
\includegraphics[width=5in]{img/ch02/Fig1_figuresupplement10.pdf}
\caption[3'-Seq reads mapping counts for the nuclear mRNA fraction ]{\textbf{3'-Seq reads mapping counts for the nuclear mRNA fraction } Total number of reads that map to the genome (mapped) and the number of final reads used for analysis that are cleanly mapped (Clean Mapped) by nuclear mRNA library. Cleanly mapped reads are reads that mapped successfully and passed the filtering for mispriming (MP) as described in the Methods.}
\label{fig:NucMapCount}
\end{figure}
\clearpage

\begin{figure}[!htb]
\centering
\includegraphics[width=5in]{img/ch02/Fig1_figuresupplement11.pdf}
\caption[3'-Seq reads mapping counts for the total mRNA fraction]{\textbf{3'-Seq reads mapping counts for the total mRNA fraction} Total number of reads that map to the genome (mapped) and the number of final reads used for analysis that are cleanly mapped (Clean Mapped) by total mRNA library. Cleanly mapped reads are reads that mapped successfully and passed the filtering for mispriming (MP) as described in the Methods.}
\label{fig:TotMapCount}
\end{figure}
\clearpage

\begin{figure}[!htb]
\centering
\includegraphics[width=5in]{img/ch02/Fig3_figuresupplement4.pdf}
\caption[Proportion of eQTLs explained by apaQTLs]{\textbf{Proportion of eQTLs explained by apaQTLs} Proportion of eQTLs putatively explained by apaQTLs separated by fraction. Expression QTLs could be explained by apaQTLs identified from both fractions. This observation is robust to apaQTL association p-value cutoffs. We observed that apaQTLs explain a slightly higher proportion of previously unexplained eQTLs. Explained/Unexplained status of each eQTL was determined previously in Li et al. 2016 \citep{li_rna_2016}}
\label{fig:popExp}
\end{figure}
\clearpage

\section{Supplementary file 1}\label{ch02-Supplementary file 1} 

\subsection{3' Sequencing of nuclear mRNA captures mRNA species independent of mRNA decay}\label{ch02-decay}

To ensure that applying 3' Seq on the nuclear mRNA fraction would reflect polyadenylation usage of transcripts that have yet to be subject to decay, we verified that the nuclear mRNA 3' Seq captures features of nascent mRNA species prior to and independent from mRNA decay. To this end, we tested whether the ratio of nuclear to total mRNA 3' Seq reads correlates with measures of RNA decay. We reasoned that if nuclear mRNA captures mRNA species before they are subject to decay, then genes with more nuclear reads relative to total reads should have higher rates of mRNA decay. We used 4sU-seq (30m) data and RNA decay measurements collected in the same panel of lymphoblastoid cell lines (LCLs) as was used in this study as a proxy for mRNA rates of decay. The RNA decay and 4sU data were originally collected and processed in Pai et al. 2012 \citep{pai_contribution_2012} and Li et al. 2016 \citep{li_rna_2016}, respectively. We further used RNA sequencing data collected in the same LCLs as used in this study and details regarding data processing can be found in Li et al. 2016 \citep{li_rna_2016}. 

We computed a score reflecting the nascent transcription rate for each gene as the normalized 4sU count over the sum of the RNA-seq and 4sU counts. This is because 4sU captures nascent mRNA that were metabolically labelled with a modified uridine. After a fixed amount of time (30min in this case), the modified transcripts are sequenced. A positive correlation between 4sU/RNA and nuclear/total 3' Seq across genes suggests that the nuclear 3' Seq captures polyadenylation usage at an earlier stage of the mRNA lifecycle. 

In Li et al 2016, the authors presented a relationship between the same nascent transcription rate and a measure relative mRNA decay rate. They reported a negative correlation between nascent transcription and relative decay, whereby genes with faster nascent transcription also show faster rates of decay. We show a similar relationship between decay rate and our ratio of nuclear 3' Seq to nuclear and total mRNA 3' Seq, suggesting that we are capturing mRNA transcripts prior to mRNA decay in the nuclear fraction. To compute the correlations, we used the summary of the lm function in R. 

Together, these correlations show that nuclear fraction 3' Seq captures information that is not captured in 3' Seq from the total mRNA fraction, and importantly, that the difference is biologically rather than technically driven. Thus, we were able to use 3' Seq data from both nuclear and total mRNA fraction to study how genetic effects regulate APA at multiple stages of the mRNA lifecycle.  In particular, the observed difference between APA in nuclear versus total mRNA fraction supports the notion that if genetic effects were detectable only in the total mRNA fraction, we should suspect that the genetic effect drives variation in post-transcriptional regulation such as decay or export. This assumption is based on the premise that mRNA from the total fraction better reflect mRNA diversity subsequent to decay and export. Because we do not see many examples of genetic effects only identified in the total mRNA fraction, we propose that nearly all genetic effect drive variation in APA co-transcriptionally. 

\begin{figure}
\centering \includegraphics[width=5in]{img/ch02/figureAppendix1.pdf}
\caption[Relationship between 3' Seq and nascent transcription]{ \textbf{Relationship between 3' Seq and nascent transcription}  {\bf (A)}Nuclear 3' Seq captures polyadenylation of nascent transcripts. The ratio of new mRNA to steady-state mRNA (x axis) are plotted against the ratio of 3' Seq reads from the nuclear fraction to 3' Seq reads from the total mRNA fraction (y axis). Slope, R2, pvalue from a linear regression.  {\bf (B)} Nuclear 3' Seq captures polyadenylation of mRNA independent of mRNA decay. The relative decay rate of each gene (x axis) are plotted against the ratio of 3' Seq reads from the nuclear fraction to 3' Seq reads from the total mRNA fraction (y axis). Slope, R2, pvalue from a linear regression.}
\label{fig:Supplementaryfile1-Fig1}
\end{figure}


\subsection{Intronic polyadenylation in other human tissues}\label{ch02-intronic-other-tissues}

In this study we used LCLs because of the rich molecular phenotyping that has been performed on the same cell lines. By collecting 3' Seq from cell nuclei we uncovered many more intronic PAS than expect. However, we are currently unable to validate whether these PAS are used in other human tissues because we are the first, to the best of our knowledge, to perform 3' Seq on mRNA from isolated nuclei in human cells.

That said, in order to estimate the extent to which intronic PAS we identified in the nuclear fraction are used in other human cell types, we turned to other APA studies that used a similar method to identify whole cell PAS. We reasoned that because total mRNA captures a small fraction of nuclear mRNA, it may be possible to use total mRNA to quantify the extent of intronic alternative polyadenylation in nuclei. For example, we found that 387 intronic PAS that were highly used in LCL nuclear mRNA were also detectable in LCL total mRNA. We can thus ask what fraction of these 387 intronic PAS also show evidence of usage in other cell-types from data collected by other studies on PAS. As baseline, we used 3' Seq usage data collected by Lianoglou et al., which include LCLs and four other cell-types (Breast, Ovary, Testes, Stem Cells). We found that about 10\% of the 387 intronic PAS showed detectable usage in total 3' seq from LCLs collected by the Lianoglou study \citep{lianoglou_ubiquitously_2013}. By contrast, around 5\% of the intronic PAS showed usage in Breast, and Testes. Usage of 3' Seq data from another study performed by Derti and colleagues suggest that nearly 10\% of the 387 PAS showed detectable usage \citep{derti_quantitative_2012}. Thus, these results suggest that there is at most a 2-fold difference in alternative polyadenylation in nuclei in other cell-types. While a 2-fold difference may appear large, we expect different cell-types to use different PAS depending on the specific genes that are expressed. 

\begin{figure}
\centering \includegraphics[width=5in]{img/ch02/figureAppendix2.png}
\caption[Intronic PAS Discovered in other tissues]{\textbf{Intronic PAS Discovered in other tissues} Intronic PAS enriched in the nuclear mRNA fraction of LCLs as detected in the total mRNA fraction of other human tissues. Barplot showing the percent of nuclear intronic PAS (of 387) discovered in whole cell 3' Seq from Derti et al. \citep{derti_quantitative_2012}, or Lianoglou et al. \citep{lianoglou_ubiquitously_2013}  Bar for each tissue is colored by study in which the data was collected.}
\label{fig:Supplementaryfile1-Fig2}
\end{figure}



\subsection{RNA binding motifs}\label{ch02-RBPs}

3' UTRs are hotspots for RNA binding protein (RBP) motifs. When bound, RBPs can affect post transcriptional gene regulatory processes such as translation efficiency and nuclear export. We wanted to investigate whether genetic variants can impact APA by affecting binding of RBPs. To do this, we asked whether 3' UTRs with an apaQTL were more likely to be bound by an RBP than expected by chance. We downloaded eCLIP data for 25 RBPs collected by the ENCODE project in human K562 cells. We identified several RBPs enriched for genes with apaQTLs associated with 3' UTR PAS, but the overall enrichments were weak and are unlikely to explain the mechanism that underlie most apaQTLs. We did not see a similar enrichment for genes with intronic PAS apaQTLs. Interestingly, we found that the RNA binding proteins with the strongest enrichments are FUS and SAFB. These are intriguing result given the known function of FUS as a splice factor that guide nuclear export. We next asked if a genetic variant could be identified as an apaQTL due to differentially effects on one isoform but not the others. While we do not expect this to be the case genome wide, we do expect a small number of examples where a QTL could affect binding of an RBP and therefore isoform-specific post-transcriptional gene regulation.  We identified 37 nuclear and 26 total apaQTLs overlapping eCLIP peaks. Of note, two apaQTLs disrupt binding for UPF1 which is a critical factor for nonsense mediated decay. A caveat to this analysis is the cell type specificity of RBP binding. eCLIP data is not available for LCLs. 


\begin{figure}
\centering \includegraphics[width=5in]{img/ch02/figureAppendix3.pdf}
\caption[Enrichment for RNA binding in K652 cells]{\textbf{Enrichment for RNA binding in K652 cells}{\bf (A)}Enrichment for K562 cell RBP binding in 3' UTRs of genes with apaQTLs most strongly associated with a PAS in 3' UTRs compared to genes without apaQTL {\bf (B)} Enrichment for K562 cell RBP binding in 3' UTRs of genes with apaQTLs most strongly associated with an intronic PAS compared to genes without apaQTL }
\label{fig:Supplementaryfile1-Fig3}
\end{figure}


\subsection{Correlation between variance in ribosome occupancy and variance in APA }\label{ch02-var-ribo}


Variation in 3' UTR length can drive variation in translation efficiency. We wanted to test if this effect can be seen at the level of inter individual variation without requiring the existence of a QTL. We reasoned that if APA plays a role in modulating translation efficiency, then we would expect a correlation between APA variance and ribosome occupancy variance. When we correlated the variance in usage for the most highly used PAS for each gene, we see a weak but significant positive correlation between APA variance and ribosome occupancy variance (Correlation = 0.15, $p <2.2\times10^{-16}$).   

\begin{figure}
\centering \includegraphics[width=5in]{img/ch02/figureAppendix4.png}
\caption[Variance in APA and Ribosome Occupancy]{\textbf{Variance in APA and Ribosome Occupancy} Individual usage variance of the most highly used PAS in each gene (x axis) correlates with individual variance in ribosome occupancy (y axis) as measured in Li et al 2016.\citep{li_rna_2016}}
\label{ffig:Supplementaryfile1-Fig4}
\end{figure} 


\subsection{Colocalization}\label{ch02-coloc}


In the main text we assert that APA can explain a proportion of the unexplained eQTLs, i.e. chromatin independent eQTLs. We primarily relied on correlation in order to draw this conclusion. However, to strengthen our claim, we used colocalization to ask if apaQTLs might generally be causal for the correlated eQTLs. To quantify the amount of colocalization between our apaQTLs and eQTLs, we used the COLOC package to test whether the apaQTL and eQTL associations share a causal SNP. The COLOC package estimates Bayes Factors for 4 alternative hypotheses. PP0: No association with either trait, PP1: No association with trait 1, PP2: No association with trait 2, PP3: Association with trait 1 and trait 2, two independent SNPs, and PP4: Association with trait 1 and trait 2, one shared SNP. If causal SNPs for an apaQTL and an eQTL is the same SNP, then PP4 is expected to be large > 0.5. One limitation of COLOC is that it is very sensitive to sample size and tend to assign large posterior probability to PP0, PP1, PP2 when either of the QTL mapping suffer from low power. This is because QTL mapping suffer from low power due to very small sample sizes compared to GWASs, for which coloc was designed for. To overcome this limitation, we used the ratio PP4/(PP3+PP4) to assess the colocalization probability instead of PP4/(PP0+PP1+PP2+PP3+PP4). To further increase power in our analysis, we used summary statistics from eQTLs identified on Geuvadis YRI LCL sample (n = 90) and used coloc to find colocalization between the eQTL signal and apaQTLs for the polyadenylation site (PAS) that is the most significant for the same gene. We expect this to be a lower bound for the actual number of colocalized eQTL-apaQTL SNPs because only one PAS for each gene is tested. Overall, we found that 33 genes had both and apaQTL and an eQTL and for which PP3+PP4 from coloc was 0.2 or greater. We found that the vast majority of genes (26, 78.8\%) had a PP4/(PP3+PP4) value greater than 0.5, which indicates that the apaQTL and eQTL are more likely to share a causal SNP than not. Thus, we conclude that most apaQTLs that are determined to be eQTLs are likely to be causal, and further likely explain all the SNP effect on gene expression.


\begin{figure}
\centering \includegraphics[width=5in]{img/ch02/figureAppendix5.png}
\caption[Colocalization of apaQTLs and eQTLs]{\textbf{Colocalization of apaQTLs and eQTLs} The apaQTL and eQTLs for the large majority of genes that have both are more likely to colocalize than not. Histogram of number of genes with an apaQTL and eQTL for different values of PP4/(PP3 +PP4).}
\label{ffig:Supplementaryfile1-Fig5}
\end{figure} 

\subsection{Evaluating the robustness of our finding to false positives caused by mispriming}\label{ch02-MP-robust}

We took various measures to ensure that misprimed reads are not included in our analysis. For example, we include filters both at the read and PAS level according to previous reports using the same experimental protocol (methods). In order to test if mispriming could still be responsible for the PAS we identified, we have looked at the base composition around our PAS. The results are below with 10 base pairs up and downstream of the PAS (PAS are at position 10 on plot). We have separated PAS based on their location and on whether the PAS is annotated in polyADB. We found a very similar base pair composition for all PAS except for intronic PAS that are unannotated in polyA DB. This suggests there may be some amount of mispriming for intronic PAS that are not annotated in the polyA DB. By quantifying the increase in A at nearby position around unannotated intronic PAS relative to annotated intronic PAS, we estimate that up to 20\% of our unannotated intronic PAS may be explained by mispriming. 


\begin{figure}
\centering \includegraphics[width=5in]{img/ch02/figureAppendix6.png}
\caption[Base Composition around PAS]{\textbf{Base Composition around PAS} Position weight matrices representing base composition 10 bps upstream and downstream of identified PAS separated by location and presence/absence of site in polyA DB.}
\label{fig:Supplementaryfile1-Fig6}
\end{figure} 


However, we believe that the vast majority of unannotated intronic PAS are likely to be real. To support this view, we found that of the 9,605 unannotated intronic PAS, 24.6\% have a canonical polyadenylation signal site upstream of the PAS. This matched the fraction of intronic PAS that are annotated, and is significantly higher than background (which is about 0.24\%). Furthermore, the location of the canonical polyadenylation signal site relative to the PAS location follows the expected distribution, which is 10-30bp upstream.   


\begin{figure}
\centering \includegraphics[width=5in]{img/ch02/figureAppendix7.png}
\caption[Signal site distribution for intronic unannotated PAS]{\textbf{Signal site distribution for intronic unannotated PAS} Stacked histogram of polyadenylation signal sites upstream of unannotated intronic PAS. Distribution similar in shape and structure to that in Figure 2.1D. }
\label{fig:Supplementaryfile1-Fig7}
\end{figure} 


While we would argue that a 20\% rate of mispriming is reasonably low, and removing more PAS would lead to many false negatives, we nevertheless decided to rerun our analysis after removing intronic PAS that have not been previously annotated, to make sure that our results are robust to misprimed contaminates. We re-calculated the correlation between intronic effect sizes and eQTL effect sizes and found that the correlation is stronger than when the unannotated PAS are included (349 vs 357). This suggests that mispriming may be increasing noise. 

\begin{figure}
\centering \includegraphics[width=5in]{img/ch02/figureAppendix8.png}
\caption[Figure 2.3A without unannotated intronic PAS]{\textbf{Figure 2.3A without unannotated intronic PAS} Scatter plot of intronic apaQTL effect sizes after removing associations with unannotated intronic PAS plotted against their eQTL effect sizes. Supplemental to Figure 2.3A. }
\label{fig:Supplementaryfile1-Fig8}
\end{figure} 



We also found that the proportion of eQTLs that are significant apaQTLs does not change dramatically (18\% vs 17.3\% of unexplained eQTLs using the 0.05 cutoff).

\begin{figure}
\centering \includegraphics[width=5in]{img/ch02/figureAppendix9.png}
\caption[Proportion eQTL explained without unannotated intronic PAS]{\textbf{Proportion eQTL explained without unannotated intronic PAS} Proportion of putatively explained by apaQTLs separated by fraction after removing associations with unannotated intronic PAS. Expression QTLs could be explained by apaQTLs identified from both fractions. This observation is robust to apaQTL association p-value cutoffs. We observed that apaQTLs explain a slightly higher proportion of previously unexplained eQTLs. Explained/Unexplained status of each eQTL was determined previously in Li et al. 2016.\citep{li_rna_2016}}
\label{fig:Supplementaryfile1-Fig9}
\end{figure} 

Lastly, we found that nearly all apaQTLs that are not eQTLs but are associated with differences in translation and protein expression are not affected by the removal of unannotated intronic PAS (20 vs 25). Together these analyses suggest that even if our set of intronic PAS include some false positives, these PAS do not drive the main conclusions of our work.  



\clearpage
\section{Supplementary Tables}\label{ch02-supplementary-tables}

\begin{table}[!htb]
\caption[Expression Independent eQTLs]{\textbf{Expression Independent eQTLs} 
(see supplementary file associated with this
  dissertation) apaQTL whose lead SNP is nominally associated with protein expression levels but not expression. Table includes p-value and slope for the associated between the lead SNP and nuclear APA usage, gene expression levels, protein expression levels, and ribosome occupancy (as measured using ribo-seq). mRNA, protein and translation data reported in Li et al. 2016\citep{li_rna_2016}.}
\label{tab:ch02-s1}
\end{table}

\begin{table}[!htb]
\caption[Meta Data]{ \textbf{Meta data}  (see supplementary file
  associated with this dissertation) Library information for each Yoruba lymphoblastoid cell line, including sample,
collection, and read information. Column names as described: Sample\_ID: Sample ID, line: YRI Line,fraction: Molecular fraction, batch:3' Sequencing batch, fqlines: number of lines in fastQ file (used to calculate reads),reads: number of sequenced reads, mapped: number of mapped reads,  Mapped\_noMP:number of reads mapped after misprimmed reads are removed, prop\_MappedwithoutMP, proportion of usable reads, Sex: Sex of YRI sample, Wake\_Up: Date up cell line wakeup, Collection: Date of cell collection, count1:cell count measurements ($1x10^{^6}$) , count2:cell count measurements ($1x10^{^6}$), alive1: percent of cells alive calculated with trypan blue stain, alive2:percent of cells alive calculated with trypan blue stain, alive\_avg: average of two percent alive measurements,  undiluted\_avg: average of two cell count measurements ($1x10^{^6}$), Extraction: Date of mRNA extraction, Concentration: RNA concentration ng/ul, ratio260\_280: RNA quality collected from nanodrop, to\_use: amount of RNA input for 3' seq, h20: amount of water used for 3' seq, threeprime\_start: data of library collection, Cq: quantification measurement from qPCR during 3'  seq library preparation, cycles: cycles used for library prep, library\_conc: concentration of 3'  seq library (ng/ul).}
\label{tab:ch02-s2}

\end{table}



